is.yaml

```diff
@@ -0,0 +1,2 @@
+lenguaje: javascript
+entidad: komando4.js

```

Comment:
Este fichero ya no existe. Una vez que lo hayáis borrado, podíais poner algún fichero que sí exista (camino completo)

Reply:
Cierto, ha sido un error

Reply:
Arrelgado en commit 391e20ea1898f7b8ec2efe02f6d6aa74fbd04894